lib/cluster.js

```diff
@@ -684,10 +684,12 @@ function workerInit() {
 
   Worker.prototype.destroy = function() {
     this.exitedAfterDisconnect = true;
-    if (!this.isConnected()) process.exit(0);
-    var exit = process.exit.bind(null, 0);
-    send({ act: 'exitedAfterDisconnect' }, () => process.disconnect());
-    process.once('disconnect', exit);
+    if (!this.isConnected()) {
+      process.exit();
```

Comment:
Sorry if there was confusion. In my last comment, I said to either get rid of the `var` completely, or make it `const`. By get rid of, I meant make the `0` exit code inline, as it was prior to this PR. Without specifying the `0`, this is a very subtle breaking change.